handsontable/src/plugins/copyPaste/copyPaste.js

```diff
@@ -626,13 +626,15 @@ export class CopyPaste extends BasePlugin {
   onCopy(event) {
     const focusedElement = this.hot.getFocusManager().getRefocusElement();
     const isHotInput = event.target?.hasAttribute('data-hot-input');
+    const selectedCell = this.hot.getSelectedRangeLast()?.highlight;
+    const TD = selectedCell ? this.hot.getCell(selectedCell.row, selectedCell.col) : null;
```

Comment:
I've found that the fix does not work for fixed rows/columns. Retrieving the topmost cell solves the issue.
```suggestion
    const selectedCell = this.hot.getSelectedRangeLast()?.highlight;
    const TD = selectedCell ? this.hot.getCell(selectedCell.row, selectedCell.col, true) : null;
```